src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -104,31 +161,16 @@
               (-> (NumberFormat.)
                   (.setType "CURRENCY"))))))))
 
-(defn date-time
-  "using the given excel date formatting string"
-  [date-time format-patterm]
-  (-> (CellData.)
-      (.setUserEnteredValue
-       (-> (ExtendedValue.)
-           ;; Heaven help us. Excel serial time and casting to a float.
-           (.setNumberValue
-            (double (+ 2 (time/in-days (time/interval (time/date-time 1900 1 1)
-                                                      date-time)))))))
-      (.setUserEnteredFormat
-       (-> (CellFormat.)
-           (.setNumberFormat
-            (-> (NumberFormat.)
-                (.setType "DATE")
-                (.setPattern format-patterm)))))))
-
 (defn row->row-data
   "google-ifies a row (list of columns) of type string?, number? keyword? or CellData."
   [row]
   (-> (RowData.)
       (.setValues (map coerce-to-cell-data row))))
 
 (defn write-sheet
-  "writes values to a specific sheet (tab). Breaks down requests into batches of ~10k cells."
+  "writes values to a specific sheet (tab). Breaks down requests into batches of ~10k cells.
+  Overwrites all the existing rows on the sheet.  Doesn't alter the number of
+  columns on the sheet and so writing more columns than the sheet has will error"
```

Comment:
As we did in the drive api, is there any utility in splitting apart requests and transactions, so that e.g. writing to multiple sheets within the same spreadsheet can occur in one http request? Does the underlying java/rest api support that, or am I just imagining?